Makefile

```diff
@@ -107,6 +107,34 @@ install:
 
 # Browser-specific hosts targets
 
+.PHONY: hosts-iridium
```

Comment:
Could you please move this block down, after `hosts-brave-user` but before `hosts-firefox`?